nix/pkgs/haskell/haskell.nix

```diff
@@ -157,13 +251,30 @@ let
           # See https://github.com/input-output-hk/iohk-nix/pull/488
           cardano-crypto-praos.components.library.pkgconfig = lib.mkForce [ [ libsodium-vrf ] ];
           cardano-crypto-class.components.library.pkgconfig = lib.mkForce [ [ libsodium-vrf ] ];
+
+          # By default haskell.nix chooses the `buildPackages` versions of these `build-tool-depeends`, but
```

Comment:
Is this a haskell.nix bug?

Reply:
I think it is because we are using `build-tool-depends` as if it was `run-tool-depends`.

Reply:
So haskell.nix is interpreting it correctly.